javascript/modules/client/src/internal/utils/plugins.ts

```diff
@@ -0,0 +1,219 @@
+import { ProposalStatus } from "../interfaces/common";
+import { Erc20Proposal, MultisigProposal, VoteOptions } from "../interfaces/plugins";
+
+// TODO: delete this file
+
+export function getERC20ProposalsWithStatus(proposals: Erc20Proposal[]) {
+  const now = new Date();
+
+  return proposals.map(proposal => {
+    if (proposal.startDate >= now) {
+      return { ...proposal, status: ProposalStatus.PENDING };
+    } else if (proposal.endDate >= now) {
+      return { ...proposal, status: ProposalStatus.ACTIVE };
```

Comment:
Same as above: Temporary code related to mock data, I would personally put it on a file clearly named as such, like temp-mock.ts or similar. Otherwise, visibility of that is much lower.